src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.ts

```diff
@@ -106,11 +107,21 @@ export class InstructorHelpStudentsSectionComponent extends InstructorHelpSectio
     ],
   }];
 
+  @Input() collapseStudentEditDetailsInChild: Subject<boolean> = new Subject<boolean>();
```

Comment:
There is no need to have this `@Input`.

Instead we can add `@Input` to `isEditDetailsCollapsed`.

We can directly pass the attribute: `<tm-instructor-help-students-section isEditDetailsCollapsed='collapseStudentEditDetailsInChild'></tm-instructor-help-students-section>`